README.md

```diff
@@ -125,6 +125,7 @@ To learn more about the docker pull rate limits and the open source software pro
 | [Snippy](https://hub.docker.com/r/staphb/snippy) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/snippy.svg?style=popout)](https://hub.docker.com/r/staphb/snippy) | <ul><li>4.4.5</li><li>4.5.1</li><li>4.6.0</li></ul> | https://github.com/tseemann/snippy |
 | [snp-dists](https://hub.docker.com/r/staphb/snp-dists) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/snp-dists.svg?style=popout)](https://hub.docker.com/r/staphb/snp-dists) | <ul><li>0.6.2</li><li>0.8.2</li></ul> | https://github.com/tseemann/snp-dists |
 | [SNP-sites](https://hub.docker.com/r/staphb/snp-sites) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/snp-sites.svg?style=popout)](https://hub.docker.com/r/staphb/snp-sites) | <ul><li>2.3.3</li><li>2.5.1</li></ul> | https://github.com/sanger-pathogens/snp-sites |
+| [SNVPhyl-tools](https://hub.docker.com/r/staphb/snvphyl-tools) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/snvphyl-tools.svg?style=popout)](https://hub.docker.com/r/staphb/snvphyl-tools) | <ul><li>1.8.2</li></ul> | https://github.com/phac-nml/snvphyl-tools |
```

Comment:
Please change
`https://img.shields.io/docker/pulls/staphb/snvphyl-tools.svg?style=popout`
to
`https://badgen.net/docker/pulls/staphb/snvphyl-tools`

See #347 for more info

Reply:
I did that, but it looks like a broken image now.

Reply:
Let me create the repo on dockerhub and then let's refresh the page